src/components/SQForm/SQFormAutocomplete.js

```diff
@@ -109,9 +201,13 @@ function SQFormAutocomplete({
   onBlur,
   onChange,
   onInputChange,
-  size = 'auto'
+  size = 'auto',
+  lockWidthToField
```

Comment:
Can we default `lockWidthToField` to be false so we don't rely on `undefined`? Once this gets to TypeScript we'd have to make this adjustment anyway